### ISSUE TITLE
How to start the development server

### ISSUE THREAD

**Issue by sr258** [Lumieducation/Lumi#201]
I don't understand how to start the development server to debug the application.

I've done this:
```sh
npm install
npm run build
npm start
```

The browser opens and loads http://localhost:3000. However, I then get the following errors:
```
Proxy error: Could not proxy request /static/js/bundle.js from localhost:3000 to http://localhost:3001/.
See https://nodejs.org/api/errors.html#errors_common_system_errors for more information (ECONNREFUSED).

Proxy error: Could not proxy request /static/js/0.chunk.js from localhost:3000 to http://localhost:3001/.
See https://nodejs.org/api/errors.html#errors_common_system_errors for more information (ECONNREFUSED).

Proxy error: Could not proxy request /static/js/main.chunk.js from localhost:3000 to http://localhost:3001/.
See https://nodejs.org/api/errors.html#errors_common_system_errors for more information (ECONNREFUSED).
```

How do I start the second (backend?) server on 3001?

**Comment by JPSchellenberg** [comment]
Right now I am starting the development-server via electron and vscode:

```json
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Debug Main Process",
            "type": "node",
            "request": "launch",
            "preLaunchTask": "build",
            "envFile": "${workspaceRoot}/.env.server",
            "console": "integratedTerminal",
            "cwd": "${workspaceFolder}",
            "runtimeExecutable": "${workspaceFolder}/node_modules/.bin/electron",
            "windows": {
                "runtimeExecutable": "${workspaceFolder}/node_modules/.bin/electron.cmd"
            },
            "args": ["."],
            "outputCapture": "std"
        }
    ]
}
```

 I will see that I put everything in `npm start` so the development is easier.

**Comment by JPSchellenberg** [comment]
Follow up: this issue was caused by the `react-dev-utils`-dependency on 10.2.1. It was resolved by using 10.1.0 instead.
